cmd/xl-v1-healing.go

```diff
@@ -116,13 +115,13 @@ func healBucket(ctx context.Context, storageDisks []StorageAPI, bucket string, w
 		go func(index int, disk StorageAPI) {
 			defer wg.Done()
 			if _, serr := disk.StatVol(bucket); serr != nil {
-				if errors.Cause(serr) == errDiskNotFound {
+				if serr == errDiskNotFound {
 					beforeState[index] = madmin.DriveStateOffline
 					afterState[index] = madmin.DriveStateOffline
 					dErrs[index] = serr
 					return
 				}
-				if errors.Cause(serr) != errVolumeNotFound {
+				if serr != errVolumeNotFound {
```

Comment:
could have been 
`if` => `else if`

Reply:
I think the existing code is cleaner to read.